models/base.py

```diff
@@ -1018,18 +1297,32 @@ def delete_from_disk_and_database(self, session=None, commit=True, remove_folder
 
         # make sure these are set to null just in case we fail
         # to commit later on, we will at least know something is wrong
-        self.md5sum = None
-        self.md5sum_extensions = None
-        self.filepath_extensions = None
-        self.filepath = None
+        try:
+            self.md5sum = None
+            self.md5sum_extensions = None
+            self.filepath_extensions = None
+            self.filepath = None
+        except:
+            pass
```

Comment:
?? why the try/except here?  Setting a bunch of fields to None isn't going to throw an exception.

Reply:
Oh, with SQLA you'd be surprised what sorts of things throw exceptions...

Reply:
I can re-run the tests without this part and see if anything shakes out